packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -651,6 +654,13 @@ class OverlayState extends State<Overlay> with TickerProviderStateMixin {
     });
   }
 
+  /// remove all OverlayEntry
+  void clear(){
```

Comment:
It's dangerous to remove an OverlayEntry that you haven't added because the entity that added the overlay entry is not informed about this. It will assume that the overlay entry still exists leading to bugs (e.g. if they try to remove this overlay entry, it will crash because it is already removed).

This doesn't seem like an API that we would want to provide.